### PR TITLE
Bonfire MDN links - Issue #255

### PR DESCRIPTION
**Pull request by aldraco**
Links for the one-flame bonfires have been added. I can continue with the rest or leave it here, up to you.

If there are no links to show, the "helpful links" header does not display.

TODO? I can also add an input on the "create bonfire" page to allow a user to add these at time of creation.

### CODE REVIEW

**Comment by terakilobyte** [comment]
I vote for inclusion. I also like your coding style @aldraco.

**Comment by BerkeleyTrue** [comment]
LGTM

**Comment by aldraco** [comment]
Thanks!

Great, I'll keep going with the rest of the bonfires.

**Comment by BerkeleyTrue** [comment]
Are you going to them here or in a separate PR?

File: controllers/bonfire.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ var _ = require('lodash'),
     debug = require('debug')('freecc:cntr:bonfires'),
     Bonfire = require('./../models/Bonfire'),
     User = require('./../models/User'),
-    resources = require('./resources');
+    resources = require('./resources')
```
Comment (elimisteve) [inline controllers/bonfire.js]: Add comma to end of line?

File: views/bonfire/show.jade [hunk 2]
```diff
@@ -143,5 +149,9 @@ block content
                     $('.btn-twitter').attr('href', url);
                 }
         );
+        var MDNlinks = !{JSON.stringify(MDNlinks)};
```
Comment (elimisteve) [inline views/bonfire/show.jade]: Interesting. What does the `!{...}` do?

File: controllers/bonfire.js (hunk 1 above)
Comment (BerkeleyTrue) [inline controllers/bonfire.js]: Yup. good catch.

File: views/bonfire/show.jade (hunk 2 above)
Comment (BerkeleyTrue) [inline views/bonfire/show.jade]: It tells jade to to turn MDNlinks into a string, then writes the output to this section of the file as text.

Comment (elimisteve) [inline views/bonfire/show.jade]: @BerkeleyTrue I get what `JSON.stringify(MDNlinks)` does, but thanks for clarifying that the `!{...}` syntax is from Jade.  I'll look into that.

Comment (aldraco) [inline views/bonfire/show.jade]: I was thinking a separate pr. Easier to merge changes w master on my local copy iteratively than all at once later on. But I'm unaware of whatever the usual protocol is around here.

**Comment by BerkeleyTrue** [comment]
@aldraco Separate PR's are fine. If you can fix the comma typo we can merge it in.

**Comment by aldraco** [comment]
Voila!

**Comment by QuincyLarson** [comment]
I've QA'd this and it is an immediate improvement on our Bonfires. I'm going to go ahead and get this into production tonight! Thanks so much @aldraco! and @BerkeleyTrue and @elimisteve for your feedback.

**Pull request merged**